docs/providers/spark_databricks.md

```diff
@@ -11,11 +11,48 @@ With Databricks, you can leverage your Databricks cluster to compute transformat
 * Databricks Cluster
 * [Remote file storage (eg. Azure Blob Storage)](azure.md)
 
-### Transformation Sources
+### Required Azure Configurations
+
+#### Azure Blob Storage
+
+If you encounter the following error, or one similar to it, when registering a transformation, you may need to disable certain default container configurations:
```

Comment:
Feel like we should frame this less like "If you hit this error do this" and more like "This is required to be set like this to run on databricks"